components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/ext/TextView.kt

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.toolbar.ext
+
+import android.view.View
+import android.widget.TextView
+
+private const val DEFAULT_FONT_PADDING = 6
+
+fun TextView.adjustMaxTextSize(heightMeasureSpec: Int) {
```

Comment:
this should either be `internal` or live in `support-ktx` :)

Also a kdoc explaining what it does would be helpful. :)

Reply:
Whoops, forgot to mention this @rocketsroger for limiting the scope! @pocmo to the rescue 👌

Reply:
Thanks!  I'll move it to support-ktx and add the kdoc.

Reply:
Heh, I meant using the `internal` but this works too. :)

Reply:
Yes, agreed.  Support-ktx will allow other components to use it too.  So I took @pocmo 's suggestion.

Reply:
`DEFAULT_FONT_PADDING` is sort of a restriction if that's the intention since other components might want different values. Maybe try another arg with `DEFAULT_FONT_PADDING` as the default?

Reply:
Actually this one is not user customizable.  We can only turn it on and off using
https://developer.android.com/reference/android/widget/TextView#setIncludeFontPadding(boolean)

Its for the accents and its always 3dp top and bottom (total to 6dp).

Reply:
I see what you're saying.  Agree that the param should have a default value.  I'll make the changes.  Thanks,